### PR TITLE
updates to allow for additional architecture support

### PR DESCRIPTION
**Pull request by braynshock**
I tested on clean ubuntu installation. Did not break on amd64 platform.
Will start work on arm64 testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ export KUBE_DNS_API='kubernetes'            ### DNS API ENDPOINT            ###
 
 Two areas should be considered for customizing the Bootkube-CI deployment. The first area that may need modified are the variables at the top of the `bootkube-up.sh` script. The second place you may want to consider is related to the deployed SDN manifests in the [deploy-sdn](./deploy-sdn) folder. Feel free to submit an issue, or contact me on [Kubernetes Slack](https://kubernetes.slack.com/) (`v1k0d3n`).
 
+## Adding additional Architectures:
+
+Right now, amd64 is currently supported, however, framework has been placed, and have started testing using arm64 (aarch64 or arm64v8[hf]). Can be adjusted in .bootkube.env.
```
Comment (v1k0d3n) [inline README.md]: should be `.bootkube_env`

File: .bootkube_env [hunk 2]
```diff
@@ -16,5 +16,6 @@ export NSERVER02='192.168.1.70'             ### THE PRIMARY DNS SERVER      ###
 export NSERVER03='8.8.8.8'                  ### THE SECONDARY DNS SERVER    ###
 export NSEARCH01='svc.cluster.local'        ### MODIFY FOR YOUR ENVIRONMENT ###
 export NSEARCH02='jinkit.com'               ### MODIFY FOR YOUR ENVIRONMENT ###
+export KUBEARCH='amd64'                     ### MODIFY FOR YOUR ARCHITECTURE###
```
Comment (v1k0d3n) [inline .bootkube_env]: i have a search and replace nit request (which i'm going to fix in other areas myself actually). so the variable visually stands out a little more, can you change `KUBEARCH` to `KUBE_ARCH`? i need to do this on a couple others, and then i'll standardize on `KUBE_` for any variable prefix going forward just to make them stand out a bit more visually.

**Review by v1k0d3n — CHANGES_REQUESTED** [review]
these are great changes/additions, and good catch on adding support for ARM. 

just have a small syntax nit on the variable to make it stand out a little better. if you're pressed for time, i can always pull in these changes and make the modification later when i sweep through the variables for better consistency.

**Comment by braynshock** [comment]
Done.

**Review by v1k0d3n — APPROVED** [review]
awesome! thanks @braynshock

**Pull request merged**